scripts/babbage/staking-example/claim-script-staking-rewards.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+set -e
+# Unofficial bash strict mode.
+# See: http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -u
+set -o pipefail
+
+
+export BASE="${BASE:-.}"
+export WORK="${WORK:-example/work}"
+export CARDANO_NODE_SOCKET_PATH="${CARDANO_NODE_SOCKET_PATH:-example/main.sock}"
+export TESTNET_MAGIC="${TESTNET_MAGIC:-42}"
+export UTXO_VKEY1="${UTXO_VKEY1:-example/utxo-keys/utxo1.vkey}"
+export UTXO_SKEY1="${UTXO_SKEY1:-example/utxo-keys/utxo1.skey}"
+export UTXO_VKEY2="${UTXO_VKEY1:-example/utxo-keys/utxo2.vkey}"
+export UTXO_SKEY2="${UTXO_SKEY1:-example/utxo-keys/utxo2.skey}"
+export UTXO_STAKING_VKEY1="${UTXO_STAKING_VKEY1:=example/utxo-keys/utxo-stake.vkey}"
+export UTXO_STAKING_SKEY1="${UTXO_STAKING_SKEY1:=example/utxo-keys/utxo-stake.skey}"
+export UTXO_STAKING_VKEY2="${UTXO_STAKING_VKEY2:=example/utxo-keys/utxo2-stake.vkey}"
+export UTXO_STAKING_SKEY2="${UTXO_STAKING_SKEY2:=example/utxo-keys/utxo2-stake.skey}"
+export PV=v1 # Plutus Script Version
```

Comment:
Some of these aren't used. It may be worth:
1. Remove any unused environment variables
2. Setting `PV` to `v2` and using it later on as you have done for other examples (or just remove it)
The same goes for `register-and-deglegate-script-staking-address.sh`